src/App.tsx

```diff
@@ -66,7 +67,13 @@ interface StyleProps {
 
 const useStyles = makeStyles((theme: Theme) => ({
   container: {
-    background: 'linear-gradient(180deg, #FBF9F9 0%, #EFF5EF 100%)',
+    backgroundColor: theme.palette.TwClrBaseGray025,
+    backgroundImage:
+      'linear-gradient(180deg,' +
+      `${hexRgb(`${theme.palette.TwClrBaseGreen050}`, { alpha: 0, format: 'css' })} 0%,` +
+      `${hexRgb(`${theme.palette.TwClrBaseGreen050}`, { alpha: 0.4, format: 'css' })} 100%)`,
```

Comment:
for some reason I couldn't get this to work well, I had to use format: 'object' and create a string from that. See https://github.com/terraware/terraware-web/blob/main/src/components/Map/PlantingSiteMap.tsx#L29 let me know if I'm doing something wrong?

Reply:
Hmm, `hexRgb(`${theme.palette.TwClrBaseGreen050}`, { alpha: 0.4, format: 'css' })` generates `rgb(214 253 229 / 40%)` in my code here.

I think I did try using `theme.palette.TwClrBaseGreen050 as string` but the typechecker didn't like that, so I reverted to the `` `${...}` `` format instead. If I recall correctly.